nipype/interfaces/mrtrix3/utils.py

```diff
@@ -183,6 +183,7 @@ class TensorMetricsInputSpec(CommandLineInputSpec):
         argstr='-value %s', desc='output selected eigenvalue(s) file')
     component = traits.List(
         [1, 2, 3],
+        usedefault=True,
```

Comment:
From the [docs](https://mrtrix.readthedocs.io/en/latest/reference/commands/tensor2metric.html):

> **-num sequence** specify the desired eigenvalue/eigenvector(s). Note that several eigenvalues can be specified as a number sequence. For example, ‘1,3’ specifies the principal (1) and minor (3) eigenvalues/eigenvectors (default = 1).

Reply:
ok, can keep just the first one